test/tests/api/redfish_1_0/event_service_tests.py

```diff
@@ -86,19 +86,28 @@ def test_get_event_subscription(self):
     def test_submit_test_event(self):
         """ Testing POST /EventService/SubmitTestEvent  """
         global task
-        server = Httpd(port=int(HTTPD_PORT), handler_class=self.EventServiceHandler)
+        # Suppose rackhd and test stack have the same localhost
```

Comment:
This test had been converted into FIT format and is running in FIT smoke.  Is something failing here that you feel it needs to be updated?